third_party/move/move-compiler/src/parser/syntax.rs

```diff
@@ -2621,6 +2678,7 @@ fn parse_address_specifier(context: &mut Context) -> Result<AddressSpecifier, Bo
 //      StructDecl =
 //          native struct <StructDefName> <Abilities>? ";"
 //        | "struct" <StructDefName> <Abilities>? "{" Comma<FieldAnnot> "}"
+//        | "struct" <StructDefName> "(" Comma<Type> ")" <Abilities>?";"
 //        | "enum" <StructDefName> <Abilities>? "{" Comma<EnumVariant> "}"
 //      StructDefName =
 //          <Identifier> <OptionalTypeParameters>
```

Comment:
Please update the `EnumVariant` definition down 2 lines, to include the positional version.

Reply:
done